orttraining/orttraining/test/python/checkpoint/orttraining_test_load_checkpoint.py

```diff
@@ -210,6 +210,104 @@ def test_load_from_distributed_zero_full_precision_into_single_node_mixed_precis
     agg_state_dict = checkpoint.aggregate_checkpoints(checkpoint_files, pytorch_format=True)
     assert_all_states_close_pytorch(agg_state_dict, model)
 
+def test_load_from_megatron_to_non_model_parallel_node(state_dict_post_checkpoint, expected_state_dict, model, checkpoint_dir):
+    # compare the expected dictionary with the aggregated state dictionary from the ORTTrainer
+    assert_all_states_close_ort(expected_state_dict, state_dict_post_checkpoint)
+
+    # TODO: aggregate checkpoints previously saved and load it into the pytorch model for comparison, 
```

Comment:
why is this a TODO? Is it not possible to have the test already?

Reply:
for adding tests that use the torch nn module for BART, we would need to add an additional dependency of fairseq on ORT tests. Additionally, to obtain this bart tiny module, we would need to pass it appropriately pre-processed data(it needs some dictionary and input data to initialize the module from what I saw) for the Translation task, which is non-trivial(no sample present on small publicly available dataset like Wikitext-2). As this was going beyond the scope and priority of this task, and we currently have existing tests that cover the loading to pytorch, this can be deferred to if/when this becomes necessary.